images/prow-tests/Dockerfile

```diff
@@ -12,42 +12,45 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/k8s-testimages/kubekins-e2e:v20210113-cc576af-master AS stable-stuff
-
-ENV DEBIAN_FRONTEND noninteractive
+# Starting from the gcloud debian image https://github.com/GoogleCloudPlatform/cloud-sdk-docker/blob/master/debian_slim/Dockerfile
 ARG GCLOUD_VERSION=368.0.0
-ARG KUBECTL_VERSION=v1.21.4
 
-# If we don't run update, the apt-get install below don't work
-RUN apt-get update
+FROM google/cloud-sdk:${GCLOUD_VERSION}-slim AS stable-stuff
 
-# gcloud
-RUN gcloud components update --version=${GCLOUD_VERSION}
+ENV DEBIAN_FRONTEND noninteractive
+
+ARG KUBECTL_VERSION=v1.21.4
+ARG BAZELISK_VERSION=v1.11.0
```

Comment:
bazel and bazelisk are not needed anymore, so we can safely remove them here and elsewhere.

Reply:
removed